src/ui/templates/results/pagination.hbs

```diff
@@ -56,7 +56,8 @@
 
     {{#each pageNumbers}}
       {{#if this.active}}
-        <span id="active-page" class="yxt-Pagination-page" aria-label="Currently on page {{../pageNumber}}">{{../pageLabel}} {{../pageNumber}}</span>
+        <span id="active-page" class="yxt-Pagination-page yxt-Pagination-active{{this.onePageDesktop}}{{this.onePageMobile}}" aria-label="Currently on page {{../pageNumber}}">{{../pageLabel}} {{../pageNumber}}</span>
```

Comment:
We try to use SUIT style classes in SDK (minus the `yxt-` prefix). Could we add modifiers instead? e.g.

`yxt-Pagination-active yxt-Pagination-active--onePageMobile yxt-Pagination-active--onePageDesktop`